aws/flatcar-linux/kubernetes/require.tf

```diff
@@ -2,24 +2,12 @@
 
 terraform {
   required_version = ">= 0.12.0"
-}
-
-provider "aws" {
-  version = "2.31.0"
-}
-
-provider "local" {
-  version = "~> 1.2"
-}
-
-provider "null" {
-  version = "~> 2.1"
-}
-
-provider "template" {
-  version = "~> 2.1"
-}
 
-provider "tls" {
-  version = "~> 2.0"
+  required_providers {
+    aws      = ">= 2.31.0"
```

Comment:
Doesn't this allow version 3.0.0? I don't think we want that, as that would mean the aws provider probably broke compatibility. Why not maintain the symbols used before?

Reply:
Oops, bad copy-paste. will fix.

Reply:
Fixed.